_field-types/metadata-fields/field-names.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: _field_names
+nav_order: 10
+has_children: false
+parent: Metadata fields
+---
+
+# Field names
+
+The `field_names` field indexes the names of fields within a document that contain non-null values. This field support the `exists` query, which identifies documents with or without non-null values for a specified field. 
+
+The `field_names` only indexes field names when both `doc_values` and `norms` are disabled for those fields. If either `doc_values` or `norms` are enabled, the `exists` query remains functional but does not rely on `field_names`.
```

Comment:
```suggestion
`_field_names` only indexes field names when both `doc_values` and `norms` are disabled for those fields. If either `doc_values` or `norms` are enabled, the `exists` query remains functional but does not rely on `field_names`.
```

Reply:
@mgodwan @rursprung Is the text at line 13 technically accurate? If not, please make suggested edit.